snapcraft.yaml

```
@@ -76,5 +76,10 @@ apps:
       - removable-media
 
 architectures:
-  - build-on: i386
   - build-on: amd64
+  - build-on: arm64
+  - build-on: armhf
+  - build-on: i386
+  - build-on: ppc64el
```

Please ignore build errors for ppc too, as it is not actively tested.
```suggestion
  - build-on: ppc64el
    build-error: ignore
```